tests/expect/test_fish_completion.exp

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+log_user 0
+set timeout 4
+
+# Under pycharm, for some unknown reason readline displays
+# the suggestions in a different order, breaking the test.
+# This enables debug output from Hydra which is easier to test
+set env(HYDRA_COMP_DEBUG) 1
+set env(INPUTRC) /dev/null
```

Comment:
since you don't have an equivalent shell function that handles HYDRA_COMP_DEBUG you don't need this.
same for INPUTRC, I think this is bash specific.